build-system/pr-check.js

```diff
@@ -207,6 +207,12 @@ function main(argv) {
     return 0;
   }
 
+  if (!(files.some(x => x === 'package.json') &&
```

Comment:
can you check my logic again. changed it to:

`files.includes('package.json') ? !files.includes('yarn.lock') : false`

we should be OK if yarn.lock is updated but package.json is not in instances where we re-pin nested deps. unlikely but i see it happen when running `yarn update dep`